test/studies/shootout/fasta/fastaredux.graph

```diff
@@ -0,0 +1,6 @@
+perfkeys: real, real, real, real, real, real
+files: fasta.25000000.dat, fasta-lines.dat, fasta-printf.dat, fasta-qio.dat, fasta-enum.dat, fastaredux-release.dat
+graphkeys: fasta, fasta-lines, fasta-printf, fasta-qio, fasta-enum, fastaredux-release
+graphtitle: Fasta-redux Shootout Benchmark (n=25,000,000)
+ylabel: Time (seconds)
+graphname: fastaredux.25000000
```

Comment:
I seem to remember someone saying recently that the "graphname" field was no longer required.  Was it @ronawho?